stdlib/test/python/test_python_to_mojo.mojo

```diff
@@ -27,41 +28,38 @@ fn test_string_to_python_to_mojo(inout python: Python) raises:
 
 fn test_range() raises:
     var array_size: PythonObject = 2
-    # CHECK: 0
-    # CHECK-NEXT: 1
+
+    # we check that the numbers appear in order
+    # and that there are not less iterations than expected
+    # by ensuring the list is empty at the end.
+    var expected = List[Int](0, 1)
     for i in range(array_size):
-        print(i)
+        assert_equal(i, expected.pop(0))
+    assert_false(expected)
 
     var start: PythonObject = 0
     var end: PythonObject = 4
-    # CHECK: 0
-    # CHECK-NEXT: 1
-    # CHECK-NEXT: 2
-    # CHECK-NEXT: 3
+    expected = List[Int](0, 1, 2, 3)
     for i in range(start, end):
-        print(i)
+        assert_equal(i, expected.pop(0))
+    assert_false(expected)
```

Comment:
**Question** Did you mean to have this assertion?  Ditto below for the `assert_false(expected)`.

Reply:
We check that the list is empty with this. That allows us to make sure that there were only 4 iterations of the loop. Not more, not less